wagtail/wagtailadmin/rich_text.py

```diff
@@ -44,15 +49,17 @@ def render(self, name, value, attrs=None):
 
     def render_js_init(self, id_, name, value):
         if self.options is not None and 'plugins' in self.options:
+            # explicit 'plugins' config passed in options, so use that
             plugin_data = self.options['plugins']
-        elif self.plugins is not None:
+        elif self.use_legacy_plugin_config:
+            # RemovedInWagtail114Warning
```

Comment:
Is there any difference between the plugins that come from the default features and the ``halloPlugins`` list?